src/PerfView/Authentication.cs

```diff
@@ -298,6 +298,8 @@ public static SymbolReaderAuthenticationHandler AddBasicHttpAuthentication(this
         private static ChainedTokenCredential CreateTokenCredential()
         {
             return new ChainedTokenCredential(
+                new EnvironmentCredential(),
```

Comment:
Rather than chaining all of these together, let's do the following.  By default, let's set this to be an `Interactive` credential and nothing else.  Then, if the user wants to override this, they can do so by specifying one or more credential types from the list of valid credentials on the command line, using a new command line argument that I'd like you to implement called `/SymbolsAuth`.  Use an enum with flags to back the set of valid values based on what is currently supported by `TokenCredential`.